common/lib/xmodule/xmodule/modulestore/tests/test_inheritance.py

```diff
@@ -0,0 +1,62 @@
+"""
+Unit tests for testing inheritance mixins
+"""
+from __future__ import absolute_import
+
+import ddt
+import unittest
+
+from mock import Mock
+from django.utils.timezone import now, timedelta
+from xblock.core import XBlock
+from xblock.fields import ScopeIds
+from xblock.runtime import DictKeyValueStore, KvsFieldData, Runtime
+from xblock.test.tools import TestRuntime
+
+from xmodule.modulestore.inheritance import InheritanceMixin
+
+
+class TestXBlock:
+    pass
+
+
+@ddt.ddt
+class TestInheritanceMixin(unittest.TestCase):
+
+    def setUp(self):
+        """
+        Create a test xblock with mock runtime.
+        """
+        runtime = TestRuntime(
+            Mock(entry_point=XBlock.entry_point), mixins=[InheritanceMixin], services={'field-data': {}}
+        )
+        self.xblock = runtime.construct_xblock_from_class(
+            TestXBlock, ScopeIds('user', 'TestXBlock', 'def_id', 'usage_id')
```

Comment:
This method requires the type of block to be created, containing the mixins identified in the runtime.